tbxforms/layout/buttons.py

```diff
@@ -54,6 +54,7 @@ class to get them to display as secondary or warning buttons.
 
     """
 
+    input_type = "button"
```

Comment:
Required because of that change brought by Crispy Forms: https://github.com/django-crispy-forms/django-crispy-forms/blob/main/crispy_forms/layout.py#L242

( [The same class in Crispy Forms v1, for comparison](https://github.com/django-crispy-forms/django-crispy-forms/blob/1.14.0/crispy_forms/layout.py#L177-L187) )